File: letterCount.py
def letter_count(text):
    max_counter = 0
    max_letter = ''
    dictio = {}
    for letter in text:
        if letter in dictio:
            dictio[letter] += 1
        else:
            dictio[letter] = 1
        if dictio[letter] > max_counter:
            max_counter = dictio[letter]
            max_letter = letter
    return (max_letter, max_counter)

print(letter_count('jfscujhnergccwudhbmgnkwjdbgkbh'))

        

File: letterCount.js
function letterCount(text) {
  let maxCounter = 0;
  let maxletter = 0;
  let dictio = {};
  for (const letter of text) {
    if (letter in dictio) {
      dictio[letter] += 1;
    } else {
      dictio[letter] = 1;
    }
    if (dictio[letter] > maxCounter) {
      maxCounter = dictio[letter];
      maxletter = letter;
    }
  }
  return [maxletter, maxCounter];
}

console.log(letterCount("kehrfiehvwihvilqevbrlewhviebvhv"));
